test/y2storage/boot_requirements_checker_ppc_test.rb

```diff
@@ -64,8 +65,22 @@
       context "if there is already a suitable PReP partition in the disk" do
         let(:prep_partitions) { [prep_partition] }
 
-        it "does not require any partition (PReP will be reused and Grub2 can handle this setup)" do
-          expect(checker.needed_partitions).to be_empty
+        context "and it is on the boot disk" do
+          let(:boot_disk) { dev_sda }
+
+          it "does not require any partition (PReP will be reused and Grub2 can handle this setup)" do
+            expect(checker.needed_partitions).to be_empty
```

Comment:
One thing came to my mind. What happens if the existing PReP partition is formatted? Checking the code I see we would try to reuse it, what I think it is wrong. If that is true, maybe we should add some kind of control to avoid to reuse formatted partitions (and the same applies to zipl and bios_boot).

Reply:
It's not formatted. Reuse means 'install grub there'. It's the same as with bios grub. If the partition already exists you have to assume it is (has been) used. There's no way to distinguish use/non-use.